### PR TITLE
feat(docs): example of multiple catalogs defined in .pyiceberg.yaml

### PR DESCRIPTION
**Pull request by jayceslesar**
Spawned from #156

### CODE REVIEW

**Review by HonahX — APPROVED** [review]
LGTM! Thanks @jayceslesar. Just a quick, non-blocking comment from my side

File: mkdocs/docs/api.md [hunk 1]
```diff
@@ -33,6 +33,20 @@ catalog:
...
+  hive:
+      uri: thrift://127.0.0.1:9083
+      s3.endpoint: http://127.0.0.1:9000
+      s3.access-key-id: admin
+      s3.secret-access-key: password
+  rest:
+    uri: https://rest-server:8181/
+    warehouse: my-warehouse
+```
+and loaded in python by calling `load_catalog`. See below for an example.
```
[the remark this answers is not in the document]
Comment (jayceslesar) [reply mkdocs/docs/api.md:48]: Yeah I think that is more explicit -- will add

**Review by Fokko — APPROVED** [review]
Thanks @jayceslesar for fixing this! This was super obvious for me, but that's mostly because I wrote the code initially, thanks for adding this example 🙌 

Thanks @HonahX for the prompt review!

**Comment by Fokko** [comment]
@jayceslesar The Markdown linter has an improvement, can you run `make lint`?

**Comment by jayceslesar** [comment]
@Fokko my org has a manual job that can be run as part of a PR/MR that can run the lint and commit the changes? Is it worth looking at adding something similar? Or stick with the pre-commit? (which I just need to get used to haha)

**Comment by Fokko** [comment]
> my org has a manual job that can be run as part of a PR/MR that can run the lint and commit the changes? Is it worth looking at adding something similar?

Ha, I'm aware of those systems, and I like them! I haven't looked into that much, but this would require committing on someone's behalf?

**Pull request merged**